shell/src/main/java/alluxio/cli/fs/command/LsCommand.java

```diff
@@ -53,17 +53,42 @@
   // Permission: drwxrwxrwx+
   public static final String LS_FORMAT_PERMISSION = "%-12s";
   public static final String LS_FORMAT_FILE_SIZE = "%15s";
-  public static final String LS_FORMAT_LAST_MODIFIED_TIME = "%24s";
+  public static final String LS_FORMAT_TIMESTAMP = "%24s";
   public static final String LS_FORMAT_ALLUXIO_STATE = "%5s";
   public static final String LS_FORMAT_PERSISTENCE_STATE = "%16s";
   public static final String LS_FORMAT_USER_NAME = "%-15s";
   public static final String LS_FORMAT_GROUP_NAME = "%-15s";
   public static final String LS_FORMAT_FILE_PATH = "%-5s";
   public static final String LS_FORMAT_NO_ACL = LS_FORMAT_FILE_SIZE + LS_FORMAT_PERSISTENCE_STATE
-      + LS_FORMAT_LAST_MODIFIED_TIME + LS_FORMAT_ALLUXIO_STATE + " " + LS_FORMAT_FILE_PATH + "%n";
+      + LS_FORMAT_TIMESTAMP + LS_FORMAT_ALLUXIO_STATE + " " + LS_FORMAT_FILE_PATH + "%n";
   public static final String LS_FORMAT = LS_FORMAT_PERMISSION + LS_FORMAT_USER_NAME
       + LS_FORMAT_GROUP_NAME + LS_FORMAT_FILE_SIZE + LS_FORMAT_PERSISTENCE_STATE
-      + LS_FORMAT_LAST_MODIFIED_TIME + LS_FORMAT_ALLUXIO_STATE + " " + LS_FORMAT_FILE_PATH + "%n";
+      + LS_FORMAT_TIMESTAMP + LS_FORMAT_ALLUXIO_STATE + " " + LS_FORMAT_FILE_PATH + "%n";
+
+  private static final Map<String, Comparator<URIStatus>> SORT_FIELD_COMPARATORS = new HashMap<>();
+
+  static {
+    SORT_FIELD_COMPARATORS.put("creationTime",
+        Comparator.comparingLong(URIStatus::getCreationTimeMs));
+    SORT_FIELD_COMPARATORS.put("inMemoryPercentage",
+        Comparator.comparingLong(URIStatus::getInMemoryPercentage));
+    SORT_FIELD_COMPARATORS.put("lastAccessTime",
+        Comparator.comparingLong(URIStatus::getLastAccessTimeMs));
+    SORT_FIELD_COMPARATORS.put("lastModificationTime",
+        Comparator.comparingLong(URIStatus::getLastModificationTimeMs));
+    SORT_FIELD_COMPARATORS.put("name",
+        Comparator.comparing(URIStatus::getName, String.CASE_INSENSITIVE_ORDER));
+    SORT_FIELD_COMPARATORS.put("path", Comparator.comparing(URIStatus::getPath));
+    SORT_FIELD_COMPARATORS.put("size", Comparator.comparingLong(URIStatus::getLength));
+  }
+
+  private static final Map<String, Function<URIStatus, Long>> TIMESTAMP_FIELDS = new HashMap<>();
+
+  static {
+    TIMESTAMP_FIELDS.put("creation", URIStatus::getCreationTimeMs);
```

Comment:
since we already have similar fields in `SORT_FIELD_COMPARATORS`, let's use `creationTime`, `lastAccessTime`, `lastModificationTime` as the options to be consistent in `TIMESTAMP_FIELDS` too?

Reply:
I usually prefer shorter arguments, but I can also see that we want to be consistent between arguments. I am fine either way.